programs/programs/il/pe/spm.py

```
@@ -48,3 +49,67 @@ class IlTanf(Tanf):
+            # Check if household has heating/cooling expenses
+            has_heating_cooling = self.screen.has_expense(["heating", "cooling"])
+            if not has_heating_cooling:
+                return 0
+
+            # Income eligible and has expenses - return hard-coded benefit
+            household_size = self.screen.household_size
+            size_key = min(household_size, 6)
+            benefit = self.benefit_amounts.get(size_key, 0)
+            return benefit
+
+        except (KeyError, Exception):
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)